docs/source/aboutcode-toolkit/home.rst

```diff
@@ -31,8 +31,8 @@ In addition, this tool is able to generate attribution notices and
 identify redistributable source code used in your project to help you comply
 with open source licenses conditions.
 
-This version of the AboutCode Toolkit follows the ABOUT specification version 3.0 at:
-https://github.com/nexB/aboutcode-toolkit/blob/develop/SPECIFICATION.rst
+This version of the AboutCode Toolkit follows the ABOUT specification version 3.2.1 at:
+https://aboutcode-toolkit.readthedocs.io/en/latest/specificaltion.html
```

Comment:
now this is

https://aboutcode-toolkit.readthedocs.io/en/latest/specification.html

Reply:
??
what do u mean?

Reply:
ah sorry
there is a typo
I fixed it now.